jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -629,6 +629,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-gci-qa-m60:
```

Comment:
@krzyzacy I don't see any `gci-qa-` jobs in that file. Can we do the move for all of them all together, at a later time?